examples/lib/defaultHtmlStepUi.js

```diff
@@ -48,7 +51,7 @@ function DefaultHtmlStepUi(_sequencer, options) {
               <div class="row step">\
                 <div class="col-md-4 details container-fluid">\
                   <div class="cal collapse in"><p>' +
-                    '<a href="https://github.com/publiclab/image-sequencer/blob/main/docs/MODULES.md#' + step.name + '-module">' + (step.description || '') + '</a>' +
+                    '<a href='+packageLink+'>' + (step.description || '') + '</a>' +
```

Comment:
The href will need quotes here
`href="' + packageLink + '>'`

Also, spaces between the `+` and other things will make it look cleaner.